block/pkg/service/volumes.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package service
+
+import (
+	"context"
+	backendpb "github.com/opensds/multi-cloud/backend/proto"
+	"github.com/opensds/multi-cloud/block/pkg/datastore/driver"
+	pb "github.com/opensds/multi-cloud/block/proto"
+	log "github.com/sirupsen/logrus"
+)
+
+func (b *blockService) ListVolumes(ctx context.Context, in *pb.VolumeRequest, out *pb.ListVolumesResponse) error {
```

Comment:
Why do we need this params list in this manner. What is the output of ListVolumes?
ListVolumesResponse can be the return value?

Reply:
changes done